include/random.h

```diff
@@ -1,20 +1,93 @@
 #ifndef GUARD_RANDOM_H
 #define GUARD_RANDOM_H
 
-extern u32 gRngValue;
-extern u32 gRng2Value;
+// The number 1103515245 comes from the example implementation of rand and srand
+// in the ISO C standard.
+#define ISO_RANDOMIZE1(val)(1103515245 * (val) + 24691)
+#define ISO_RANDOMIZE2(val)(1103515245 * (val) + 12345)
+
+/* Notes about new functions:
```

Comment:
I think it might be good to avoid "new". Perhaps "HQ", or maybe just describe the API?

My gripe with "new" is that more functions could be added in the future, and also that functions like `RandomUniform` are new from the perspective of vanilla pokeemerald, so I think readers could find it unclear.

Reply:
Maybe something like “HQ_RANDOM support functions”?

Reply:
Sounds good to me :)